src/naive/date.rs

```diff
@@ -629,22 +629,28 @@ impl NaiveDate {
     ///
     /// If the day would be out of range for the resulting month, use the last day for that month.
     ///
-    /// Returns `None` if the resulting date would be out of range.
-    ///
     /// ```
     /// # use chrono::{NaiveDate, Months};
     /// assert_eq!(
     ///     NaiveDate::from_ymd(2022, 2, 20).checked_sub_months(Months::new(6)),
     ///     Some(NaiveDate::from_ymd(2021, 8, 20))
     /// );
     /// ```
+    ///
+    /// Returns `None` if the resulting date would be out of range.
```

Comment:
I'd prefer to keep this explanation before the examples (and I think that's common style for rustdoc).